common/src/main/scala/react/reflex/ReflexWithHandle.scala

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package react.reflex
+
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+
+import scalajs.js
+
+trait ReflexEvents extends js.Object
+
+object ReflexWithHandle {
+  // Yes, this is a hack. We use it because react-reflex expects a ReactHandle to be the
```

Comment:
This sounds good, do you think there is a risk that react-reflex may deprecate this in the future?

Reply:
There's a chance. It doesn't have a lot of stars and forks but it has 15 contributors and it seems actively maintained for the moment.